fio_graphs.py

```diff
@@ -72,6 +72,19 @@ def _parse_dir(self):
 
     def _parse_file(self, path):
         with open(path) as file_:
+            # fio adds a bunch of crud to the start of its json output. Skip it,
+            # otherwise json.load() will fail
+            while True:
+                pos = file_.tell()
+                maybe_json = file_.readline()
+                if maybe_json == None or maybe_json.startswith("{"):
```

Comment:
maybe_json == None doesn't properly detect EOF (readline returns an empty string)...
I'm going to change this to check whether readline() advanced the current offset. If it didn't then we've hit EOF.